rtsp_test.go

```diff
@@ -0,0 +1,321 @@
+package viamrtsp
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"image"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/bluenviron/gortsplib/v4"
+	"github.com/bluenviron/gortsplib/v4/pkg/base"
+	"github.com/bluenviron/gortsplib/v4/pkg/description"
+	"github.com/bluenviron/gortsplib/v4/pkg/format"
+	"github.com/bluenviron/gortsplib/v4/pkg/rtptime"
+	"github.com/bluenviron/mediacommon/pkg/codecs/h264"
+	"go.viam.com/test"
+	"go.viam.com/utils"
+
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/rimage/transform"
+)
+
+func TestRTSPCamera(t *testing.T) {
```

Comment:
Should we be running this as apart of the CI flow?

Reply:
Yes, I was going to add a ticket to do so after this is merged

Reply:
https://viam.atlassian.net/browse/RSDK-7316